### PR TITLE
AAP-15796

### PR DESCRIPTION
**Pull request by EMcWhinn**
Editing Managing containers in private automation hub

[HubDocathon] Managing content Chapter 3

https://issues.redhat.com/browse/AAP-15796

Affects `titles/hub/managing-content`

### CODE REVIEW

File: downstream/modules/hub/proc-add-container-readme.adoc [hunk 1]
```diff
@@ -27,7 +27,7 @@ Add a README to your container repository to provide instructions to your users
 
 . Navigate to menu:Execution Environments[].
 . Select your container repository.
-. On the *Detail* tab, click btn:[Add].
+. On the *Details* tab, click btn:[Add].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-add-container-readme.adoc:30]: Hey @EMcWhinn , On the Automation Hub UI, it is the Detail tab.  See screen the capture attached and let me know if you have questions:
Screenshot from 2023-09-11 10-00-51

File: downstream/modules/hub/proc-create-credential.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ To pull container images from a password or token-protected registry, create a c
 
 .Procedure
 . Navigate to {ControllerName}.
-. In the side-menu bar, click menu:Resources[Credentials].
+. In the navigation panel, click menu:Resources[Credentials].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-create-credential.adoc:11]: change click to select in this instance

File: downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc [hunk 3]
```diff
@@ -19,12 +18,14 @@ As an {PlatformNameShort} user, you can now confirm if an EE/Container is alread
         automationhub_container_signing_service_script: _path/to/executable_
 
 
-. Navigate to automation hub.
+. Navigate to {HubName}.
 
-. In the navigation pane, select menu:Signature Keys[].
+. In the navigation panel, select menu:Signature Keys[].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc:23]: I don't see this selection in the UI. We'll need to verify whether this procedure is still valid, and if so, how is it done in the new UI?

File: downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc [hunk 4]
```diff
@@ -3,11 +3,10 @@
 
 = Deploying your system for container signing
 
-Execution Environments are container images used by Ansible {ControllerName} to run jobs. This content can be downloaded to the private automation hub, and published within your organization.
+{HubNameStart} implements image signing so you can rely on better security for the EE container images.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc:6]: I don't believe EE is an approved acronym and the appropriate attribute should be used in this instance.

File: downstream/modules/hub/proc-add-group-to-container-repo.adoc [hunk 5]
```diff
@@ -13,7 +13,8 @@ Provide access to your container repository for users who need to work the image
 
 . Navigate to menu:Execution Environments[].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-add-group-to-container-repo.adoc:14]: This should be updated based on UI changes and I believe it should be as follows but should be verified with SMEs:
From the navigation panel, select menu:Execution Environments[Remote registries].
Click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Edit*.

File: downstream/modules/hub/proc-add-group-to-container-repo.adoc [hunk 6]
```diff
@@ -13,7 +13,8 @@ Provide access to your container repository for users who need to work the image
...
 . Select your container repository.
-. Click btn:[Edit] at the top right of your window.
+. Select the btn:[More Actions] icon *{MoreActionsIcon}*.
+. Click btn:[Edit].
 . Under *Groups with access*, select a group or groups to grant access to.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-add-group-to-container-repo.adoc:18]: I don't see the Groups with access selection in the UI

File: downstream/assemblies/hub/assembly-managing-containers-hub.adoc [hunk 7]
```diff
@@ -1,13 +1,12 @@
 ifdef::context[:parent-context: {context}]
 
 [id="managing-containers-hub"]
-= Managing containers in private automation hub
+= Managing containers in [PrivateHubName}
```
Comment (dcdacosta) [inline downstream/assemblies/hub/assembly-managing-containers-hub.adoc:4]: This should be {PrivateHubName} but you have a square bracket in front which is breaking the attribute.

File: downstream/modules/hub/proc-adding-an-execution-environment.adoc [hunk 8]
```diff
@@ -1,26 +1,26 @@
 
 [id="adding-an-execution-environment"]
 
-= Adding an execution environment
+= Adding an {ExecEnvShort}
 
 .Procedure
-. Navigate to menu:Execution Environments[].
+. Navigate to menu:Execution Environments[Add execution environment].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-adding-an-execution-environment.adoc:7]: This should be:
Navigate to menu:Execution Environments[Execution Environments].
Click btn:[Add execution environment].

File: downstream/modules/hub/proc-pushing-container-images-from-your-local.adoc [hunk 9]
```diff
@@ -33,27 +32,24 @@ embedded.
 --tls-verify=false
 ----
 +
-. Navigate to the automation hub and click on *Execution Environments* if that
-window is not open.
+. Navigate to the {HubName} and click on *Execution Environments* if that window is not open.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-pushing-container-images-from-your-local.adoc:35]: I would reword this to:
Log in to your automation hub.
From the navigation panel, select menu:Execution Environments[Execution Environments].

File: downstream/modules/hub/proc-sync-image.adoc [hunk 10]
```diff
@@ -13,24 +13,30 @@ You must have permission to view and pull from a private container repository.
...
-advisable to set a rate limit under Advanced Options.
-
++
+[NOTE]
+====
+Some container registries are aggressive with rate limiting. 
+It is advisable to set a rate limit under Advanced Options.
+====
++
 . Navigate to menu:Execution Environments[Execution Environments].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-sync-image.adoc:30]: Suggest rewording to the following for consistency across the library:
From the navigation panel, select ...

File: downstream/modules/hub/proc-sync-image.adoc [hunk 11]
```diff
@@ -13,24 +13,30 @@ You must have permission to view and pull from a private container repository.
...
 
 . Click *Add execution environment* in the page header.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-sync-image.adoc:32]: This should be Click btn:[Add execution environment].

**Review by dcdacosta — COMMENTED** [review]
@EMcWhinn A few minor changes needed; otherwise, LGTM!

File: downstream/assemblies/hub/assembly-delete-container.adoc [hunk 12]
```diff
@@ -6,13 +6,21 @@ ifdef::context[:parent-context: {context}]
...
-include::hub/proc-delete-container.adoc[leveloffset=+1]
+.Prerequisites
+* You have permissions to manage repositories.
 
-.Verification
-* Return to the *Execution Environments* list view. The container repository should be removed from the list.
+.Procedure
+. Navigate to {HubName}.
+. From the navigation panel, select menu:Execution Environments[Execution Environments].
+. On the container repository you would like to delete, click the btn:[More Actions] icon *{MoreActionsIcon}*, then click btn:[Delete].
```
Comment (ianf77) [inline downstream/assemblies/hub/assembly-delete-container.adoc:18]: would like -> want

File: downstream/modules/hub/con-container-registry.adoc [hunk 13]
```diff
@@ -6,4 +6,5 @@
 
 [role="_abstract"]
 
-The {HubName} container registry is used for storing and managing container images. Once you have built or sourced a container image, you can push that container image to the registry portion of private {HubName} to create a container repository.
+The {HubName} container registry is used for storing and managing container images. 
+Once you have built or sourced a container image, you can push that container image to the registry portion of private {HubName} to create a container repository.
```
Comment (ianf77) [inline downstream/modules/hub/con-container-registry.adoc:10]: When you have built

File: downstream/modules/hub/proc-add-container-readme.adoc [hunk 14]
```diff
@@ -17,15 +17,15 @@ The `context` attribute enables module reuse. Every module ID includes {context}
 
 
 [role="_abstract"]
-Add a README to your container repository to provide instructions to your users for how to work with the container. {HubNameStart} container repositories support Markdown for creating a README. By default, the README will be empty.
+Add a README to your container repository to provide instructions to your users on how to work with the container. {HubNameStart} container repositories support Markdown for creating a README. By default, the README will be empty.
```
Comment (ianf77) [inline downstream/modules/hub/proc-add-container-readme.adoc:20]: default, the README is empty.

File: downstream/modules/hub/proc-add-user-to-group.adoc [hunk 15]
```diff
@@ -19,5 +19,4 @@ You can add users to groups when creating a group or manually add users to exist
 . Navigate to the menu:Users[] tab, then click btn:[Add].
 . Select users to add from the list and click btn:[Add].
 
-
-You have added the users you selected to the group. These users now have permissions to use {HubName} assigned to the group.
+Once you add the selected users to the group, they have permissions to use {HubName} assigned to the group.
```
Comment (ianf77) [inline downstream/modules/hub/proc-add-user-to-group.adoc:22]: When you add

File: downstream/modules/hub/proc-sync-image.adoc [hunk 16]
```diff
@@ -13,24 +13,32 @@ You must have permission to view and pull from a private container repository.
...
+Some container registries are aggressive with rate limiting. 
+It is advisable to set a rate limit under Advanced Options.
+====
++
+. From the navigation panel, select menu:Execution Environments[Execution Environments].
+
+. Click btn:[Add execution environment] in the page header.
+
+. Select the registry you wish to pull from. 
+The "name" field displays the name of the image that will show up as on your local registry.
```
Comment (ianf77) [inline downstream/modules/hub/proc-sync-image.adoc:35]: of the image displayed on your local registry (?)

**Review by ianf77 — COMMENTED** [review]
A few comments, word changes. Ignore them if you want.

**Review by ianf77 — APPROVED** [review]
LGTM
